setup/action.yml

```diff
@@ -11,9 +11,9 @@ inputs:
   account-id: 
     required: false
     description: Overide the cloud account id from the default. Required when on gcp where this would be the project name. 
-  gcp_credentials: 
+  credentials: 
     required: false
-    description: GCP credentials json. Required when on gcp.
```

Comment:
Caps on GCP for consistency?

Reply:
This is what I changed it from. See the new changes. It doesn't even have this anymore.

Reply:
face palm ofc